### PR TITLE
Initial solution for lotto

### PR DESCRIPTION
**Pull request by reginafcompton**

### CODE REVIEW

**Comment by reginafcompton** [comment]
@hancush - could you review this PR, which contains code for sending emails to the lotto winners?

File: lots_admin/management/commands/send_emails.py [hunk 1]
```diff
@@ -51,6 +55,39 @@ def add_arguments(self, parser):
 
 
     def handle(self, *args, **options):
+        if options['lotto_winner_email']:
+            with connection.cursor() as cursor:
+                query = '''
+                    SELECT status.id, status.lot_id
```
Comment (hancush) [inline lots_admin/management/commands/send_emails.py]: Just to confirm (hypothetically): If an applicant was part of and subsequently won two lotteries, we want to notify that applicant in two separate emails, and that's what we are achieving here, right?

Comment (reginafcompton) [reply lots_admin/management/commands/send_emails.py]: Yep, some applicants won multiple lotteries, and they should get an individual email for each lot that won. (The lucky dogs.)

Comment (hancush) [reply lots_admin/management/commands/send_emails.py]: [You get a lot! And you get a lot!](https://media.giphy.com/media/11jENMgBweUYzS/giphy.gif)

**Pull request merged**